src/enable_variable_ops/ngraph_rewrite_pass.cc

```diff
@@ -249,17 +249,41 @@ class NGraphEncapsulationPass : public NGraphRewritePass {
       return Status::OK();
     }
 
+    // Get backend + its configurations
+    // to be attached to the nodes
+    // Precedence Order: Env Variable > BackendManager
+    std::unordered_map<std::string, std::string> config_map;
+    string backend_name = BackendManager::GetCurrentlySetBackendName();
+    const char* ng_backend_env_value = std::getenv("NGRAPH_TF_BACKEND");
+    if (ng_backend_env_value != nullptr) {
+      string backend_env = std::string(ng_backend_env_value);
+      if (backend_env.empty() ||
+          !BackendManager::IsSupportedBackend(backend_env)) {
+        return errors::Internal("NGRAPH_TF_BACKEND: ", backend_env,
+                                " is not supported");
+      }
+      backend_name = backend_env;
```

Comment:
Add a V(0) log item indicating that we what backend is specified and whether we are overriding using the environment variable.

Reply:
We do have log indicating which backend was set here: https://github.com/tensorflow/ngraph-bridge/pull/103/files/5a85aded0c924619dab17f22ccf11a3a9706a84b#diff-f3f34e20694334a7d37b19708a2ef258R272
I will change the log level and add the log for whether it was set using env variable.

Reply:
@avijit-nervana V(0) might be too much, do you think we need that or is V(1) ok?

Reply:
V(0) for backend_name, and v(1) for who is setting the backend